_data/simple-icons.json

```diff
@@ -745,6 +750,11 @@
             "hex": "1E8CBE",
             "source": "https://automattic.com/press"
         },
+        {
+            "title": "Groupon",
+            "hex": "82B742",
```

Comment:
The official color - both from their brand playbook and from their website - is `#53A318`